lib/stomp_parser.dart

```diff
@@ -213,10 +229,10 @@ class StompParser implements Parser {
         .replaceAll(RegExp(r'\r'), '\\r');
   }
 
-  Map<String, String> _escapeHeaders(Map<String, String> headers) {
+  Map<String, String> _escapeHeaders(Map<String?, String> headers) {
```

Comment:
I might be blind, but why is this signature needed? Isn't this only ever called with headers filled from the StompFrame and the field inside StompFrame has a non-null signature.

Reply:
You are right, this does not need to be nullable - just an oversight!